sklearn/metrics/_classification.py

```diff
@@ -1925,10 +1925,19 @@ def balanced_accuracy_score(y_true, y_pred, *, sample_weight=None, adjusted=Fals
     Returns
     -------
     balanced_accuracy : float
+        Balanced accuracy score.
 
     See Also
     --------
-    recall_score, roc_auc_score
+    average_precision_score : Compute average precision (AP) from prediction
+        scores.
+    precision_score : Compute the precision. The precision is the ratio
```

Comment:
No need to give the definition of the precision and recall score. Having the link available will redirect the user on the API page of these functions where they will find the definition.